tests/rootless.sh

```diff
@@ -55,6 +55,21 @@ function disable_idmap() {
 	[ -e /usr/bin/newgidmap ] && mv /usr/bin/{,unused-}newgidmap
 }
 
+# FEATURE: Opportunistic cgroups support, allowing a rootless container to set
+#          resource limits on condition that cgroupsPath is set to a path the
+#          rootless user has permissions on.
+
+function enable_cgroup() {
+	# Set up cgroups for use in rootless containers.
+	mkdir -p /sys/fs/cgroup/{blkio,cpu,cpuacct,cpuset,devices,freezer,hugetlb,memory,net_cls,net_prio,openrc,perf_event,pids,systemd}/runc-cgroups-integration-test
```

Comment:
We might want to source this list from `/proc/self/cgroups`, but I can do that in a follow-up if you prefer.

Reply:
Sure, will leave that up to you to decide.